### DISCUSSION TITLE
Release Information

### DISCUSSION THREAD

**Discussion by james-d-elliott** [authelia/authelia#6707] [General]
What information is there about the upcoming release?

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

The upcoming Authelia version 4.38 is highlighted as a significant release with several new features and improvements to existing ones. This release is described as one of the biggest to date for Authelia, incorporating major roadmap items. Despite the anticipation for new features, the development team has taken extra time to ensure the release's quality, focusing on adding automated tests for new features in both the backend and frontend. This approach aims to maintain a high-quality user experience.

The release notes mention that while the team expects a few bugs, none are anticipated to be major. The larger scale of this release means that some experimentation from users might be requested, especially those comfortable with backing up their databases. Users are encouraged to stay tuned to the chat for opportunities to assist with testing.

An important note from the release details is that some changes in version 4.38 will deprecate older configurations. Although these changes are intended to be backward compatible, users are advised to update their configurations to access new features fully. Guides and an FAQ to assist with these adjustments are planned to be published on the blog.

For those interested in trying out the pre-production builds of 4.38.0, it's strongly recommended to back up proxy configurations, Authelia configurations, and Authelia databases before proceeding. The beta version, 4.38.0-beta3, includes most, if not all, final features, with the possibility of additional features depending on time constraints. Feedback is sought through GitHub Discussions for those who wish to be involved in the beta testing process.

This comment was generated by [Glime](https://glimelab.ai/).
